### DISCUSSION TITLE
how to install this package as non-bundled version?

### DISCUSSION THREAD

**Discussion by mowses** [couds/react-bulma-components#369] [Q&A]
I would like to install `react-bulma-components` in my dev project and the installed version must contain the source files and not only the bundled files.
I mean, I am developing an app that should read the prop types of all bulma components (using https://github.com/diegohaz/parse-prop-types). but my actual installation of react-bulma-components does not have the props. `Form.Control.propTypes`, `Form.Input.propTypes` are all `undefined`.

For example: this file contains the info I need: https://github.com/couds/react-bulma-components/blob/master/src/components/button/button.js#L81-L146

Could anyone point me how to install the package without having to fork it?

**Comment by kennethnym** [comment]
You can try to install this library by [specifying the git link to this repo](https://stackoverflow.com/questions/17509669/how-to-install-an-npm-package-from-github-directly).

**Reply by mowses** [reply]
I did it like in the link you specified, but the installation still not bring the source files I need.

**Reply by kennethnym** [reply]
Yeah, my bad. Instead you should include this repo as a git submodule. That should bring all files in this repo to your project locally.
